components/air_gapped_self_hosted_purchase_modal/index.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Modal} from 'react-bootstrap';
```

Comment:
Hey apologies i know this is in draft state. We are actually moving towards using components (albiet few) from `common components`. And we happen to have Modal in `packages/components`. Using that component gives one flexibility to not change much when components are commonly done in future. Because the components under neath might change but not the API of component itself. Because we might sun set react-bootstrap in future, having Modal from `components` will not have us change this file or anything. :)